test/drivers/second_quantization/hdf5d/test_driver_hdf5.py

```diff
@@ -40,7 +43,10 @@ def test_convert(self):
         legacy_file_path = self.get_resource_path(
             "test_driver_hdf5_legacy.hdf5", "drivers/second_quantization/hdf5d"
         )
-
+        msg_convert_ref = (
+            "The convert method is deprecated as of version 0.4.0 and "
+            + "will be removed no sooner than 3 months after the release."
```

Comment:
Is the `+` even necessary here? I think Python will do automatic string concatenation